Notification icon: App crash when publishing a post

When I try to publish a post on older Android versions, the app crashes due to missing resource for the notification button:

```
03-24 15:29:19.593 1565-1565/? E/StatusBar: couldn't inflate view for notification org.wordpress.android/0xbb
                                            android.content.res.Resources$NotFoundException: File res/drawable/ic_share_white_24dp.xml from drawable resource ID #0x7f0201bd
                                                at android.content.res.Resources.loadDrawable(Resources.java:1923)
                                                at android.content.res.Resources.getDrawable(Resources.java:659)
                                                at android.widget.TextView.setCompoundDrawablesWithIntrinsicBounds(TextView.java:1826)
                                                at android.widget.RemoteViews$TextViewDrawableAction.apply(RemoteViews.java:1181)
                                                at android.widget.RemoteViews.performApply(RemoteViews.java:2169)
                                                at android.widget.RemoteViews.apply(RemoteViews.java:2129)
                                                at android.widget.RemoteViews$ViewGroupAction.apply(RemoteViews.java:1113)
                                                at android.widget.RemoteViews.performApply(RemoteViews.java:2169)
                                                at android.widget.RemoteViews.apply(RemoteViews.java:2129)
                                                at com.android.systemui.statusbar.BaseStatusBar.inflateViews(BaseStatusBar.java:582)
                                                at com.android.systemui.statusbar.BaseStatusBar.addNotificationViews(BaseStatusBar.java:750)
                                                at com.android.systemui.statusbar.phone.PhoneStatusBar.addNotification(PhoneStatusBar.java:732)
                                                at com.android.systemui.statusbar.CommandQueue$H.handleMessage(CommandQueue.java:272)
                                                at android.os.Handler.dispatchMessage(Handler.java:99)
                                                at android.os.Looper.loop(Looper.java:137)
                                                at android.app.ActivityThread.main(ActivityThread.java:4745)
                                                at java.lang.reflect.Method.invokeNative(Native Method)
                                                at java.lang.reflect.Method.invoke(Method.java:511)
                                                at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786)
                                                at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553)
                                                at dalvik.system.NativeStart.main(Native Method)
                                             Caused by: org.xmlpull.v1.XmlPullParserException: Binary XML file line #1: invalid drawable tag vector
                                                at android.graphics.drawable.Drawable.createFromXmlInner(Drawable.java:877)
                                                at android.graphics.drawable.Drawable.createFromXml(Drawable.java:818)
                                                at android.content.res.Resources.loadDrawable(Resources.java:1920)
                                                at android.content.res.Resources.getDrawable(Resources.java:659) 
                                                at android.widget.TextView.setCompoundDrawablesWithIntrinsicBounds(TextView.java:1826) 
                                                at android.widget.RemoteViews$TextViewDrawableAction.apply(RemoteViews.java:1181) 
                                                at android.widget.RemoteViews.performApply(RemoteViews.java:2169) 
                                                at android.widget.RemoteViews.apply(RemoteViews.java:2129) 
                                                at android.widget.RemoteViews$ViewGroupAction.apply(RemoteViews.java:1113) 
                                                at android.widget.RemoteViews.performApply(RemoteViews.java:2169) 
                                                at android.widget.RemoteViews.apply(RemoteViews.java:2129) 
                                                at com.android.systemui.statusbar.BaseStatusBar.inflateViews(BaseStatusBar.java:582) 
                                                at com.android.systemui.statusbar.BaseStatusBar.addNotificationViews(BaseStatusBar.java:750) 
                                                at com.android.systemui.statusbar.phone.PhoneStatusBar.addNotification(PhoneStatusBar.java:732) 
                                                at com.android.systemui.statusbar.CommandQueue$H.handleMessage(CommandQueue.java:272) 
                                                at android.os.Handler.dispatchMessage(Handler.java:99) 
                                                at android.os.Looper.loop(Looper.java:137) 
                                                at android.app.ActivityThread.main(ActivityThread.java:4745) 
                                                at java.lang.reflect.Method.invokeNative(Native Method) 
                                                at java.lang.reflect.Method.invoke(Method.java:511) 
                                                at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786) 
                                                at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553) 
                                                at dalvik.system.NativeStart.main(Native Method) 
03-24 15:29:19.643 3109-3109/? E/AndroidRuntime: FATAL EXCEPTION: main
                                                 android.app.RemoteServiceException: Bad notification posted from package org.wordpress.android: Couldn't expand RemoteViews for: StatusBarNotification(pkg=org.wordpress.android id=187 tag=null score=0 notn=Notification(pri=0 contentView=org.wordpress.android/0x1090071 vibrate=null sound=null defaults=0x0 flags=0x10 kind=[null] 1 action))
                                                     at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1361)
                                                     at android.os.Handler.dispatchMessage(Handler.java:99)
                                                     at android.os.Looper.loop(Looper.java:137)
                                                     at android.app.ActivityThread.main(ActivityThread.java:4745)
                                                     at java.lang.reflect.Method.invokeNative(Native Method)
                                                     at java.lang.reflect.Method.invoke(Method.java:511)
                                                     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786)
                                                     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553)
                                                     at dalvik.system.NativeStart.main(Native Method)
```


### Steps to reproduce the behavior

1. Start a new post
2. Type something
3. Tap on the Publish button
4. The app crashes

##### Tested on Emulator, Android 16 and 18

--- a/WordPress/src/main/java/org/wordpress/android/ui/posts/services/PostUploadNotifier.java
+++ b/WordPress/src/main/java/org/wordpress/android/ui/posts/services/PostUploadNotifier.java
@@ -131,7 +131,7 @@ public class PostUploadNotifier {
             shareIntent.putExtra(Intent.EXTRA_SUBJECT, post.getTitle());
             PendingIntent pendingIntent = PendingIntent.getBroadcast(mContext, 0, shareIntent,
                     PendingIntent.FLAG_CANCEL_CURRENT);
-            notificationBuilder.addAction(R.drawable.ic_share_white_24dp, mContext.getString(R.string.share_action),
+            notificationBuilder.addAction(R.drawable.ic_share_24dp, mContext.getString(R.string.share_action),
                     pendingIntent);
         }
         doNotify(notificationId, notificationBuilder.build());

